web/src/app/WebApp/Installers/Nextcloud/NextcloudSetup.php

```diff
@@ -51,6 +51,7 @@ public function install(array $options = null): bool {
 				"maintenance:install",
 				"--database mysql",
 				"--database-name " . $this->appcontext->user() . "_" . $options["database_name"],
+				"--database-host " . $options["database_host"],
```

Comment:
this should also be
```
"--database-host " . \Hestiacp\quoteshellarg\quoteshellarg($options["database_host"]),
```
But again the same issue is all over the place here, so you don't need to fix it in this PR if you don't want to